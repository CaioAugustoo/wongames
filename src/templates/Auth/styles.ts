import styled from 'styled-components'

export const Wrapper = styled.main``

export const BannerBlock = styled.div``

export const Subtitle = styled.h3``

export const Footer = styled.p``

export const Content = styled.div``
