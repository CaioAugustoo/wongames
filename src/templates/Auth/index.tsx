import * as S from './styles'

const Auth = () => (
  <S.Wrapper>
    <h1>Auth</h1>
  </S.Wrapper>
)

export default Auth
