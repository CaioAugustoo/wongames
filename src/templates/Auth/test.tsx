import { render, screen } from '@testing-library/react'

import Auth from '.'

describe('<Auth />', () => {
  it('should render the heading', () => {
    const { container } = render(<Auth />)

    expect(screen.getByRole('heading', { name: /Auth/i })).toBeInTheDocument()

    expect(container.firstChild).toMatchSnapshot()
  })
})
