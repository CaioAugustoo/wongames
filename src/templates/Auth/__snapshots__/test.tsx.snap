// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Auth /> should render the heading 1`] = `
<main
  class=""
>
  <h1>
    Auth
  </h1>
</main>
`;
